package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "coffechat",
+  "version": "0.0.0",
+  "description": "Connect Mentors and Mentees by making the process of connecting and then scheduling appointments as easy as possible.",
```

Comment:
Could probably use a catchier description here. @pooja-malpani do you have anything on hand?